gsutil.py

```diff
@@ -42,6 +42,28 @@ def OutputAndExit(message):
   sys.exit(1)
 
 
+def _fix_google_module():
+  """Reloads the google module to prefer our third_party copy.
+
+  When python is not invoked with the -S option, it can preload google module
+  via .pth file. After this happens, our third_party google
+  package may not in the __path__. After our third_party dependency directory is
+  put at the first place in the sys.path, google module should be reloaded,
+  so that our vendored copy can be preferred.
+  """
+  if 'google' not in sys.modules:
+    return
+  # This is a hacky solution as reloading a module is never recommended.
+  # But given that it is rare that users will run into this issue where
+  # they might have google-auth already installed in their python environment,
```

Comment:
Why not just put all this information in the docstring instead of half-docstring and half-code comment?
Also, "Python" should be capitalized here.

Reply:
Done